benchmark/BDN.benchmark/SpanVsPointer.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Runtime.CompilerServices;
+using BenchmarkDotNet.Attributes;
+
+namespace BDN.benchmark
+{
+    public unsafe class SpanVsPointer
```

Comment:
I would suggest adding a comment that briefly summarizes the benchmarks. Something like this:

```suggestion
    /// <summary>
    /// Benchmarks to quantify the impact of using pointer vs. span accesses.
    /// </summary>
    public unsafe class SpanVsPointer
```

Reply:
perhaps in a later PR as this just merged :)